src/applications/edu-benefits/components/ReviewCardField.jsx

```diff
@@ -319,6 +319,9 @@ export default class ReviewCardField extends React.Component {
       this.props.formContext.onError();
     } else {
       this.setState({ editing: false, canCancel: true });
+      if (typeof get('onSaveClick', this.props.uiSchema) === 'function') {
```

Comment:
The `get` isn't really doing anything here, you could get rid of it and just do `typeof this.props.uiSchema.onSaveClick`

Reply:
Would it matter that this would record the submission-successful event twice?